### PR TITLE
remove unused strings (Thanks to kevoutings for providing all the unused strings)

### PR DESCRIPTION
**Pull request by ryuk201198**

### CODE REVIEW

**Comment by Tommy-XL** [comment]
It's used strings
Screenshot_20240313_211358
Screenshot_20240313_211553

**Comment by Tommy-XL** [comment]
It' also used
Screenshot_20240313_211831

**Comment by ryuk201198** [comment]
> It's used strings Screenshot_20240313_211358 Screenshot_20240313_211553

Btw why aren't we using Prefix.Addon for these like other addons?

**Comment by Ultradragon005** [comment]
> Btw why aren't we using Prefix.Addon for these like other addons?

frrr

File: Resources/Lang/en_US.json [hunk 1]
```diff
@@ -2529,14 +2426,11 @@
...
   "SureUse.quit": "We will kick you and block you from entering this lobby again. This setting is irreversible. If you really want it, please send the command /qt {0}",
   "PlayerIdList": "List of player IDs: ",
-  "turnOn": "On",
-  "turnOff": "Off",
   "CancelStartCountDown": "The starting countdown was cancelled",
   "RestTOHESetting": "TOHE settings have been restored to default",
   "FPSSetTo": "FPS Set To: {0}",
   "HostKillSelfByCommand": "The lobby Host decided to commit suicide",
   "SyncCustomSettingsRPC": "Synchronized RPC",
-  "TOHEVisitorCount": "To date TOHE has been used <color={0}>{1}</color> times~",
```
Comment (Ultradragon005) [inline Resources/Lang/en_US.json:2539]: ohhh the og TONX days

**Pull request merged**